auth_signup_verify_email/controllers/main.py

```diff
@@ -15,41 +15,71 @@
 
 
 class SignupVerifyEmail(AuthSignupHome):
+
     @http.route()
     def web_auth_signup(self, *args, **kw):
-        if (http.request.params.get("login") and
-                not http.request.params.get("password")):
-            return self.passwordless_signup(http.request.params)
+        if (request.params.get("login") and
+                not request.params.get("password")):
+            return self.passwordless_signup(request.params)
         else:
             return super(SignupVerifyEmail, self).web_auth_signup(*args, **kw)
 
+    @property
+    def msg_invalid_email_error(self):
+        return _("That does not seem to be an email address.")
+
+    @property
+    def msg_something_wrong_error(self):
+        return _("Something went wrong, please try again later or contact us.")
+
+    @property
+    def msg_check_inbox_info(self):
+        return _("Check your email to activate your account!")
+
     def passwordless_signup(self, values):
         qcontext = self.get_auth_signup_qcontext()
 
         # Check good format of e-mail
         if not validate_email(values.get("login", "")):
-            qcontext["error"] = _("That does not seem to be an email address.")
-            return http.request.render("auth_signup.signup", qcontext)
+            qcontext["error"] = self.msg_invalid_email_error
```

Comment:
and this makes your "change that message" task easier? :smile: 

In any case if you are customizing stuff in you own addons you'll get translations for it especially if you have a multilang website (you have all your translations in one place).
Plus, if you have only one lang, it's even faster do handle it like this then handling extra translations. 

Personally I don't like to change a message via translations, sounds like a dirt&quick fix. But it's my idea :stuck_out_tongue_winking_eye:

Reply:
Downloading a `.pot` of a couple of addons and putting translations in a folders might not be what you desire, but at the end of the day, it's a very easy task to do (it can be even done through GUI by your translation team) and if we follow that logic, we'd have to create a method for every single translation in Odoo. But it turns out that method already exists: `_()`

Besides, while that system exists, now if I have to create a sub-addon of this one, I have 2 ways of doing the same thing: subclassing and doing the `i18_extra` thing. Which one to use? Maybe subclassing for this addon and `i18n_extra` for all other addons? Why the difference? It's the typical case of reinventing the wheel. You'd possibly end up just using `i18n_extra` since it works the same all across Odoo. [The Zen of Python](https://www.python.org/dev/peps/pep-0020/#the-zen-of-python) says:

> There should be one-- and preferably only one --obvious way to do it.

I think that principle applies here.

So, again, if there's a standarized way of doing things, while working in community I still think it's better to follow the standard than to fight it.

Reply:
I strongly disagree here: mocking a translation instead of changing the original according to your needs is not to follow the zen of python. Plus, you can still do that via UI, Plus 2, this change does not make the module more complex, the contrary.
But it's my personal POV, the module is yours and I don't want to spend more time on this: rolling back :stuck_out_tongue_winking_eye: